biome.json

```diff
@@ -48,10 +43,7 @@
     "enabled": true,
     "lineWidth": 120,
     "indentStyle": "space",
-    "formatWithErrors": true,
```

Comment:
I removed the ignore and reran the formatter. It didn't result to any fixes so I think we're good to remove it.

Reply:
Oh because of https://github.com/webpro-nl/knip/commit/b346903187cebf19fc0516d2ea46bd8dec7d4dfe, it now fails :D